tests/cases/dom/traverse/next.js

```diff
@@ -0,0 +1,57 @@
+QUnit.module(
+    "next()",
+    {
+        beforeEach: function ()
+        {
+            document.getElementById("qunit-fixture").innerHTML =
+                '<div class="test-element-next element1"></div>' +
+                '<div class="test-element-next element2"></div>' +
+                'Some text' +
+                '<div class="test-element-next element3"></div>';
+        }
+    }
+);
+
+
+QUnit.test(
+    "next() with an element in the middle, without selector",
+    function (assert)
+    {
+        const next = mojave.dom.traverse.next;
+        const element = document.querySelector(".test-element-next.element2");
+
+        const result = next(element);
+
+        assert.ok(result, "found 1 element");
+        assert.ok(result.classList.contains("element3"), "contains .element3");
+    }
+);
+
+
+QUnit.test(
+    "next() with an element at the end, without selector",
+    function (assert)
+    {
+        const next = mojave.dom.traverse.next;
+        const element = document.querySelector(".test-element-next.element3");
+
+        const result = next(element);
+
+        assert.notOk(result, "found 0 elements");
```

Comment:
I would like to explicitly test for `assert.ok(null === result` as it is stated that `null` is the return type.